app/models/event.rb

```diff
@@ -1,6 +1,5 @@
 class Event < ApplicationRecord
-  belongs_to :place
+  belongs_to :place 
   belongs_to :user
-
-  validates :name, name_lenght: { minimum: 5 }
```

Comment:
Why did you remove this validation? If I'm right it should be on develop already.